cmake/FindAvroCpp.cmake

```diff
@@ -0,0 +1,63 @@
+#[=======================================================================[.rst:
+FindAvroCpp
+-----------
+
+Finds Avro C++. Does not (currently) support version matching.
+
+IMPORTED Targets
+^^^^^^^^^^^^^^^^
+
+The following :prop_tgt:`IMPORTED` targets may be defined:
+
+``Avro::AvroCpp``
+  AvroCpp library.
+``Avro::avrogencpp``
+  avrogencpp command-line executable.
+
+Result variables
+^^^^^^^^^^^^^^^^
+
+This module will set the following variables in your project:
+
+``AvroCpp_FOUND``
+  true if AvroCpp headers, library, and executable were found
+``AVROCPP_INCLUDE_DIR``
+  the directory containing AvroCpp headers
+``AVROCPP_LIBRARY``
+  AvroCpp library to be linked
+``AVROCPP_AVROGEN_EXECUTABLE``
+  path to avrogencpp tool
+
+TODO
+^^^^
+
+* Components
+  * library
+  * executable
```

Comment:
Are you planning to resolve the TODO in this PR too?

Reply:
I'll probably do this when I move the find module into the irods repository.